source/Xunit.Gherkin.Quick/CoreModel/FeatureFile/FeatureFile.cs

```diff
@@ -13,11 +13,16 @@ public FeatureFile(GherkinDocument gherkinDocument)
             GherkinDocument = gherkinDocument ?? throw new System.ArgumentNullException(nameof(gherkinDocument));
         }
 
-        public global::Gherkin.Ast.Scenario GetScenario(string scenarioName)
+        public global::Gherkin.Ast.ScenarioDefinition GetScenario(string scenarioName)
         {
-            return GherkinDocument.Feature.Children.FirstOrDefault(s => s.Name == scenarioName) as global::Gherkin.Ast.Scenario;
+            return GherkinDocument.Feature.Children.FirstOrDefault(s => s.Name == scenarioName);
         }
 
+		public global::Gherkin.Ast.ScenarioDefinition GetBackgroundScenario()
```

Comment:
Return type for this method should be `Background`, just like return type of the next method below is `ScenarioOutline`.